spring-boot-conventions/reference/CONVENTIONS.md

```diff
@@ -17,6 +19,37 @@ The submitted pod intent can be:
           image: springio/petclinic
   ```
 
+Most of the Spring Boot conventions will either modify or add properties to the environment variable `JAVA_TOOL_OPTIONS`.
+
+If a `JAVA_TOOL_OPTIONS` property already exists for a workload, the convention will use the existing value rather than the value the convention has been designed to apply by default (The provided property value will be used for the pod spec mutation.)
+
+<a id="set-java-tool-options-property"></a>**Instructions to set a `JAVA_TOOL_OPTIONS` property for a workload:**
+
+There are two approaches:
+1. Set property using the `--env` flag:<br/>
+When creating/updating a workload, a `JAVA_TOOL_OPTIONS` property can be set using the Tanzu CLI `apps` plugin as follows: 
+    ```
+    tanzu apps workload create APP-NAME --env JAVA_TOOL_OPTIONS="-D[property]=[value]"
+    ```
+    For example, setting the management port to 8080 rather than the [spring-boot-actuator-convention](#spring-boot-actuator-convention) default (port 8081) can be achieved via the following Tanzu CLI command:
+    ```
+    tanzu apps workload create APP-NAME --env JAVA_TOOL_OPTIONS="-Dmanagement.server.port=8080"
+    ```
+1. Set property using `workload.yaml`:<br/>
+If desired, the JAVA_TOOL_OPTIONS value(s) can be provided in the workload.yaml as follows:
+    ```
+    apiVersion: carto.run/v1alpha1
+    kind: Workload
+    ...
+    spec:
+     env:
+     - name: JAVA_TOOL_OPTIONS
+      value: -Dmanagement.server.port=8082
```

Comment:
can we add 1 more space here to be on the same level as `name`?

Reply:
good catch! - 
yes the "v" in "value" should be aligned with the "n" in "- name:" in the line above.